src/tir/ir/data_layout.cc

```diff
@@ -350,20 +350,16 @@ Array<PrimExpr> BijectiveLayout::BackwardShape(const Array<PrimExpr>& shape) con
                         self->src_layout->axes, self->backward_rule);
 }
 
-BijectiveLayout BijectiveLayoutNode::make(const Layout& src_layout,
-                                          const Layout& dst_layout) {
+BijectiveLayout::BijectiveLayout(Layout src_layout, Layout dst_layout) {
   auto n = make_object<BijectiveLayoutNode>();
 
-  n->src_layout = src_layout;
-  n->dst_layout = dst_layout;
+  n->src_layout = std::move(src_layout);
+  n->dst_layout = std::move(dst_layout);
 
-  if (!GetStoreRule(&n->forward_rule, n->src_layout, n->dst_layout)) {
-    // not convertible
-    return BijectiveLayout();
+  if (GetStoreRule(&n->forward_rule, n->src_layout, n->dst_layout)) {
```

Comment:
@yzhliu @anijain2305 please take a look here

Reply:
add a comment: to be consistent with old behavior, create a nullptr layout when argument is invalid